Networking/SR Linux/ci/Jenkinsfile

```
@@ -1,5 +1,46 @@
+def update_topology() {
+    def version = formatVersion()
+    if (version) {
+        def yamlFile = readYaml file: "Networking/SR Linux/containerlab/topology.yml"
+        yamlFile['topology']['nodes']['inmanta-server']['image'] = "ghcr.io/inmanta/orchestrator:${version}"
+        writeYaml file:"Networking/SR Linux/containerlab/topology.yml",data: yamlFile, overwrite: true
+    }
+}
+
+
+def formatVersion() {
+    def version = env.rc_version
```

Did you test this file?

```suggestion
    def version = env.version
```

---

yes I did. I decided to rename it at the last moment but forgot to rename it at all the other locations. sorry about that